internal/vsp/feepayment.go

```diff
@@ -137,6 +137,48 @@ func parseTicket(ticket *wire.MsgTx, params *chaincfg.Params) (
 	return
 }
 
+// calcHeights checks if the ticket has been mined, and if so, sets the live
+// height and expiry height fields. Should be called with mutex already held.
+func (fp *feePayment) calcHeights() {
+	_, minedHeight, err := fp.client.wallet.TxBlock(fp.ctx, &fp.ticketHash)
+	if err != nil {
+		// This is not expected to ever error, as the ticket has already been
+		// fetched from the wallet at least one before this point is reached.
+		fp.client.log.Errorf("Failed to query block which mines ticket: %v", err)
```

Comment:
i would add an explicit return here instead of assuming minedHeight will be zero and fall into the next if statement